core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -355,6 +363,11 @@ oppia.factory('ExplorationPlayerService', [
             interactionRulesService));
         var answerIsCorrect = classificationResult.outcome.labelledAsCorrect;
 
+        if (answerIsCorrect &&
+            PlayerCorrectnessFeedbackEnabledService.isEnabled()) {
```

Comment:
So, the correctCount is incremented only when the `PlayerCorrectnessFeedbackEnabledService.isEnabled()` is true.

Can this be turned to true after the user has correctly attempted some questions? If yes, which count would you want to show - total number of correct answers or the number of correct answers after enabling flag?

Instead, wouldn't it be better to increase correctCount at every correct answer and show only if the flag is enabled?

Reply:
It can be turned true, but the user will have to reload the page(for 'PlayerCorrectnessFeedbackEnabledService' to come into effect). And on reloading, the user will have to start from the beginning.
Therefore,  increasing correctCount at every correct answer and showing it only if the flag is enabled would also produce the same effect.